ghostjs-examples/test/network_throttling_test.js

```diff
@@ -0,0 +1,43 @@
+import ghost from 'ghostjs';
+import assert from 'assert';
+
+import localServer from './fixtures/server';
+
+describe('Network throttling', () => {
+
+  before(localServer)
+  after(localServer.stop)
+
+  it ('Takes longer to load a page on a slow connection', async () => {
+    // This test should just run in a Chrome environment
+    if (ghost.testRunner.match(/chrome/)) {
+      let options = {
+        networkOptions: {
+          connectionType: ghost.networkTypes['2g'],
+          uploadThroughput: 50,
+          downloadThroughput: 50,
+        }
+      }
+
+      const NS_PER_S = 1e9
+      const url = 'http://localhost:8888/basic_content.html'
```

Comment:
(even though I'm not really a fan of going to the real network, I think it might make sense in this case.)

Though ideally we might spin up a webserver that returns a stream of a huge amount of bytes (so we don't need to check in a large file into the codebase).